deluge/core/torrentmanager.py

```diff
@@ -1349,5 +1351,10 @@ def torrents_status_update(self, torrent_ids, keys, diff=False):
         else:
             # Ask libtorrent for status update
             self.torrents_status_requests.insert(0, (d, torrent_ids, keys, diff))
-            self.session.post_torrent_updates()
+            if lt_1_1_compat:
+                # libtorrent 1.1 takes a combination of status_flags_t`s here
+                # like torrent_handle::status()
+                self.session.post_torrent_updates(0xffffffff)
```

Comment:
Yes looks like a bug in the [binding](https://github.com/arvidn/libtorrent/blob/RC_1_1/bindings/python/src/session.cpp#L775)

This should fix it:
```
-    .def("post_torrent_updates", allow_threads(&lt::session::post_torrent_updates))
+   .def("post_torrent_updates", allow_threads(&lt::session::post_torrent_updates), arg("flags") = 0xffffffff)
```